tests/e2e/test_api.py

```diff
@@ -98,11 +123,12 @@ def check_content_type(response, content_type, message=""):
     assert response.headers["content-type"].startswith(content_type), message
 
 
+@retry(max_attempts=3, wait_between_runs=10)
 def test_readiness():
     """Test handler for /readiness REST API endpoint."""
     endpoint = "/readiness"
     with metrics_utils.RestAPICallCounterChecker(metrics_client, endpoint):
-        response = client.get(endpoint, timeout=BASIC_ENDPOINTS_TIMEOUT)
+        response = client.get(endpoint, timeout=LLM_REST_API_TIMEOUT)
```

Comment:
looks like you need to rebase your branch to get rid of some of these spurious changes.